cadc-storage-adapter-fs/src/main/java/org/opencadc/inventory/storage/fs/OpaqueFileSystemStorageAdapter.java

```diff
@@ -165,6 +164,7 @@ public class OpaqueFileSystemStorageAdapter implements StorageAdapter {
     final Path contentPath;
     private final int bucketLength;
     private final List<Namespace> preservationNamespaces = new ArrayList<>();
+    private final List<Namespace> purgeNamespaces = new ArrayList<>();
```

Comment:
Not a Set? Same comment for preservationNamespaces.

Reply:
Namespace doesn't implement Comparable and I didn't want to go modify that library; it's harmless to use list and allow duplicates in this case